### DISCUSSION TITLE
How about integrating plugin for resolving merge conflict?

### DISCUSSION THREAD

**Discussion by lianghx-319** [LazyVim/LazyVim#305] [Q&A]
I found that it is not easy from VSCode to neovim for newer.

Is there a 「Best Practice」to resolve merge conflict in neovim?

**Comment by dpetka2001** [comment]
Every tool has a learning curve. You should just take your time reading the docs and familiarizing yourself with the tool. You can't expect to jump from one tool to another and everything works the same way. The best way according to my experience is to start fresh and then add little by little if you really want to learn the tool.

Edit: Oh you meant git merge conflicts. I thought conflicts from transitioning from vscode to neovim. My bad, apologies.

**Comment by folke** [comment]
I personally just use lazygit, but a lot of people use https://github.com/sindrets/diffview.nvim

**Reply by aortizoj15** [reply]
I have not used diffview but can recommend lazygit as well from experience. [Here](https://github.com/jesseduffield/lazygit/blob/assets/resolving-merge-conflicts.gif) is a gif of how resolving merge conflicts looks. It makes it fairly easy and I love that it is integrated to LazyVim by using `<leader>gg` keymap.
